kuma/static/styles/components/syntax/_base.scss

```diff
@@ -52,7 +52,7 @@ Enhancements!
 .token.url,
 .language-css .token.string,
 .style .token.string {
-    background-color: transparent;
+    background: none!important;
```

Comment:
Hey @Jokerwin! Thank you so much for your contribution 🎉 

As far as possible we want to avoid the use of `!important`(you will notice that stylelint has failed in CI for specifically this reason), it is generally bad practice(although sometimes it cannot be avoided) and causes more pain down the line :) The tricky bit here is to ensure our code overrides what Prism is doing irrespective of the cascade.

To accomplish that without using `!important` we need to increase the specificity of our piece of code. Thankfully, the classes here will always be used on a `span` element and so, I would ask that we do the following instead.

```
span.token.url,
.language-css span.token.string,
.style span.token.string {
    background-color: transparent;
}
```

Reply:
Hey @schalkneethling thank you so much for your review! I agree with you about `!importat` to avoid the use of this so I've updated the code with the suggested changes.